deliverables/ruby_monk_problems/array_of_fixnums.rb

```diff
@@ -1,3 +1,3 @@
-def array_of_fixnums?(array)
-  # Write your code here
+def non_duplicated_values(values)
```

Comment:
Looks like you meant to accidentally copied this in here instead of `array_of_fixnums?`